src/private/files.ts

```diff
@@ -487,11 +487,17 @@ export namespace files {
    * @private
    * Hide from docs
    *
-   * Open download preference folder
+   * Open download preference folder or filePath if its value is non-null
+   * @param filePath: Path of the file which should be opened
```

Comment:
This is very confusing. Will this API be opening the file? If so, that's a huge security risk.

Reply:
Depending on the **filePath** parameter value, if it is null, it will open the download preference folder configured in Teams client and if non null it will open the folder in which file is present which is provided as filePath param.

Reply:
The API will open the File Path in one of the scenario as mentioned in above comment, can you elaborate more on the security risk, if any. And how can i go about this feature if there is a security risk.